apps/src/lib/kits/maker/PlaygroundComponents.js

```diff
@@ -18,6 +18,7 @@ import TouchSensor from './TouchSensor';
 import Piezo from './Piezo';
 import NeoPixel from './NeoPixel';
 import Led from './Led';
+import experiments from '../../../util/experiments';
```

Comment:
We don't have a common standard.  I've been using relative paths within apps/src and only using the `@cdo/apps` alias any place tests refer to source files.  This is mostly because the JetBrains refactoring tools don't work particularly well with the alias - WebStorm sees it as a link into node_modules, and RubyMine doesn't know what to do with it at all.